connection.go

```diff
@@ -873,3 +878,49 @@ func (c *Connection) closeNetwork() {
 func (c *Connection) getLastActivityTime() time.Time {
 	return time.Unix(0, c.lastActivity.Load())
 }
+
+func (c *Connection) sendBufSize() (int, int, error) {
+	if c.sysConn == nil {
+		return 0, 0, fmt.Errorf("no sys call conn")
```

Comment:
nit: `errors.New` when there is no arg?

Reply:
Yep, leftover from my PoC. Created a top-level error using `errors.New` and returning that instead.